script.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+spatch -sp_file cocci/assignment_as_value.cocci tests/data/inputs/assignment_as_value.c > cocci_outputs/assignment_as_value.csv
```

Comment:
Oh I see that being able to test the patch individually could be in need. In that case, should I just try to save the output in memory (if I am able to make it work) instead of creating an actual file. In other words, right now we don't need to see actual output, but need to know whether the test failed and what it failed？